### PR TITLE
[WIP] Update documentation

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,59 +1,28 @@
 Kebechet
 --------
 
-  I'm Kebechet, goddess of freshness. I will keep your dependencies fresh and up-to-date.
+  I'm Kebechet, goddess of freshness. I will keep your repository and dependencies fresh and up-to-date.
```
Comment (goern) [inline README.rst]: s/repository/sources/ ??

File: README.rst [hunk 2]
```diff
@@ -1,59 +1,28 @@
...
-=========================================================
-
-To use Kebechet with the old fashion ``requirements.in`` and ``requirements.txt`` files, commit ``requirements.in`` file into the root of your Git repository structure. Kebechet will automatically pin down packages for you and create an initial pull request with ``requirements.txt``. File ``requirements.in`` should state your direct dependencies and version specification you expect for dependency solver to be used during dependency resolution (you can also add restrictions for your indirect dependencies there to avoid updates of transitive dependencies introducing bugs). File ``requirements.txt`` is automatically managed by Kebechet and it will produce fully pinned down application stack for your application.
-
-Managing development requirements using a separated ``requirements.in`` is currently not supported.
-
 Configuration of Kebechet
 =========================
 
-Kebechet is configured using a simple YAML configuration file. In this configuration file you state which repositories you wish to keep updated. An example of a YAML configuration file that monitors Kebechet itself is shown below:
```
Comment (goern) [inline README.rst]: why simple?

File: README.rst [hunk 3]
```diff
@@ -1,59 +1,28 @@
...
+Kebechet is configured using a simple YAML configuration file. Check documentation of managers to get all the available options.
 
-.. code-block:: yaml
+Each codfigration entry configures a manager. Please check `managers and examples <https://github.com/thoth-station/kebechet/tree/master/kebechet/managers>`_ to get info about configuration options.
```
Comment (goern) [inline README.rst]: second word

File: README.rst [hunk 4]
```diff
@@ -1,59 +1,28 @@
...
 
-    update:
-      - slug: thoth-station/kebechet  # GitHub slug in form of <github-org>/<github-repo>
-        label:
-          - bot
-          - kebechet
+The YAML configuration file can be supplied directly as a path to a file on filesystem as well as as a URL to a file - handy for managing configuration of your Kebechet deployment in a Git repository (you have to supply a URL to a raw YAML configuration file).
```
Comment (goern) [inline README.rst]: s/as as/as/

File: README.rst [hunk 5]
```diff
@@ -1,59 +1,28 @@
...
 
-The ``label`` configuration option states a list of labels that should be applied to pull requests or issues that are automatically raised by Kebechet.
+Managers
+========
 
-The YAML configuration file can be supplied directly as a path to a file on filesystem as well as as a URL to a file - handy for managing configuration of your Kebechet deployment in a Git repository (you have to supply a URL to a raw YAML configuration file).
+A list of managers with their configuration (documentation lives in a manger directory) can be found `in the repo <https://github.com/thoth-station/kebechet/tree/master/kebechet/managers>`_.
```
Comment (goern) [inline README.rst]: s/in a manger/in the manager's/

File: README.rst [hunk 6]
```diff
@@ -1,59 +1,28 @@
...
 
 Issues created by Kebechet
 ==========================
 
-If there are any issues that have serious impact on Kebechet functionality, Kebechet will automatically open an issue in the given repository. These issues can be configuration issues of Kebechet itself, issues in your dependency management such as errors in dependency management files or issues with package retrieval (such as misconfiguration in Python indexes, resolving issues or package version removal).
+If there are any issues that have serious impact on Kebechet functionality, Kebechet will automatically open an issue in the given repository. These issues can be configuration issues of Kebechet itself or issues on manager when there are issues in the manager itself.
```
Comment (goern) [inline README.rst]: this section is pretty... confusing, can you rephrase the second sentence?

File: kebechet/managers/README.rst [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+        from kebechet.utils import cloned_repo
+
+        with cloned_repo(self.service_url, self.slug) as repo:
+            with open('my_file.txt', 'w') as my_file:
+                my_file.write("Hello, Kebechet!")
+
+            repo.git.add(my_file)
+            repo.git.push()
+
+The last thing you need to do, is to register your manager to `REGISTERED_MANAGERS` so Kebechet knows about your manager. Manager can be referenced by its name in lowercase (class name without the "manager" suffix).
```
Comment (goern) [inline kebechet/managers/README.rst]: "via `REGISTERED_MANAGERS`" rather then "to"?

File: kebechet/managers/update/README.rst [hunk 8]
```diff
@@ -1,11 +1,79 @@
 Kebechet Update Manager
 -----------------------
 
-This manager is responsible for updates of packages in a repository.
+This manager is responsible for automatic updates of packages in a repository based on `Pipfile` or `requirements.txt`
```
Comment (goern) [inline kebechet/managers/update/README.rst]: with packages you mean dependencies?

File: kebechet/managers/update/README.rst [hunk 9]
```diff
@@ -1,11 +1,79 @@
...
+      # servie_url: <URL>
+      # tls_verify: true/false
+      managers:
+        - name: update
+          configuration:
+            labels:
+              # Labels for opened issues and pull requests.
+              - bot
+
+You can find this manager in action `here <https://github.com/thoth-station/kebechet/pull/46>`_, `here <https://github.com/thoth-station/kebechet/pull/85>`_ or `here <https://github.com/thoth-station/solver/issues/38>`_.
```
Comment (goern) [inline kebechet/managers/update/README.rst]: s/find/see/

File: README.rst (hunk 2 above)
Comment (fridex) [reply README.rst]: This part was completely removed.
